### DISCUSSION TITLE
Publish app to Marketplace - Question About GitHub App Listing Review Process Timeline

### DISCUSSION THREAD

**Discussion by johnnyfish** [community/community#145962] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi everyone, I recently submitted my app for listing on the GitHub Marketplace, ensuring all the required information and criteria were met. Currently, my listing is in the stage:
"Your listing is under review by GitHub for publishing. We'll notify you via email with updates."

I couldn't find any clear references or SLAs that outline how long this review process typically takes. Can anyone share their experiences or shed some light on the approximate duration for this stage?

Additionally, in case the process takes longer than expected, what would be the best way to follow up with GitHub for an update?

Thanks in advance for any insights or advice!

![Snip20241201_37](https://github.com/user-attachments/assets/603ff1b9-d90e-4d6c-8dfc-823f12834ba7)

**Comment by GabiGrin** [comment]
How long have you been waiting for it so far?

**Reply by johnnyfish** [reply]
> How long have you been waiting for it so far?

It has been 17 days since November 14, 2024.

**Comment by Ghat0tkach** [comment]
@johnnyfish what’s the status? Did it get published? I’m facing the same issue

**Reply by johnnyfish** [reply]
Hey @Ghat0tkach! Thanks for asking. Yes, it got published - [https://github.com/marketplace/chartdb](https://github.com/marketplace/chartdb) 🎉

**Reply by Ghat0tkach** [reply]
Congratulations @johnnyfish !

**Comment by queenofcorgis** [comment, marked as answer]
Thanks everyone for the patience, we do not have a SLA in place here.

**Reply by hburaky** [reply]
Hi queenofcorgis, We are waiting for the last 2 months to publish. Is it possible to help us?

**Comment by safaii-wabam** [comment]
How long does it currently take to get published in the marketplace? We have been under review for a while. I was hoping that the process has accelerated since the last posts in this forum.
